staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -184,6 +184,7 @@ func NewCmdGet(parent string, f cmdutil.Factory, streams genericclioptions.IOStr
 	cmd.Flags().BoolVar(&o.IgnoreNotFound, "ignore-not-found", o.IgnoreNotFound, "If the requested object does not exist the command will return exit code 0.")
 	cmd.Flags().StringVar(&o.FieldSelector, "field-selector", o.FieldSelector, "Selector (field query) to filter on, supports '=', '==', and '!='.(e.g. --field-selector key1=value1,key2=value2). The server only supports a limited number of field queries per type.")
 	cmd.Flags().BoolVarP(&o.AllNamespaces, "all-namespaces", "A", o.AllNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
+	cmd.Flags().StringVar(&o.Namespace, "namespace", o.Namespace, "List options from the provided namespace. If not supplied, objects from your current namespace will be listed.")
```

Comment:
AFAIK, namespace flag is populated by for all commands already and it is kind of a standard flag. I think, it might not be correct way to add in here.